pages/common/nc.md

```diff
@@ -16,11 +16,11 @@
 
 - Serve a file:
 
-`nc -l {{port}} < {{file}}`
+`nc {{ip_address}} {{port}} < {{file}}`
```

Comment:
These appear to work for me. What seems to be the issue you're experiencing?

`-l` causes `nc` to **l**isten on the specified port number and serve the standard input. Without it, `nc` does the opposite: It connects to a listener and downloads content as a client, which it outputs to the standard output (you can also send content to the listening server via the standard input, IIRC).

Reply:
I think there's been a misunderstanding since to "receive" a file one could either listen for it or get it from a server which is serving it. Either of the following options works:

1. Serve with `nc -l port < file`, receive with `nc server_addr port > file`.
2. Wait for the file with `nc -l port > file`, send with `nc client_addr port < file`.

The second one is what I usually do too. I usually *wait for a file* with the `-l` option and have the other party *send it to me* with the classic `nc addr port < file`. That's also more secure IMHO since you cannot mistakenly send the file to someone else, while using the first option anyone could connect to you and get the file.

So yeah, that's basically what @lucianoq was thinking when changing this page. ¯\_(ツ)_/¯